packages/kbn-health-gateway-server/src/kibana/routes/root.ts

```diff
@@ -112,8 +114,9 @@ export class RootRoute implements ServerRoute {
     }
   }
 
-  private async fetch(url: string) {
-    const { protocol } = new URL(url);
+  private async fetch(host: string) {
+    const url = new URL(host);
+    appendStatusApiPath(url);
```

Comment:
AFAICT this seems accurate, I'll defer this to Luke for final +1.

Reply:
This makes sense, my only question is whether we should be allowing requests to `/api/status` to go through without the prefix. With this implementation, if someone hits `/api/status`, the gateway will transform it to `/api/status/api/status` which will 404.

The primary reason we changed the health gateway to return requests from `/` was so that the health check could match what control plane is doing today. So if we are going to be asking control plane to ultimately update all health checks to use `/api/status`, would we want to support that here as well so we don't need to come back to it later?

Reply:
>   if someone hits /api/status, the gateway will transform it to /api/status/api/status which will 404

Not sure to understand what you mean by that. 

The healthcheck handler is exclusively listening on `/`. Trying to access `/api/status` against the health gateway will be a 404 **at the gateway level**, (which will not execute the `/` handler).

So do you mean that:
1. we should add an additional `/api/status` endpoint on the gateway with the same handler as `/`
or
2. that if a a `host` is configured in the gateway's config with `/api/status` at the end, we would then not append it a second time (e.g `http://localhost:5061/somepath/api/status` would remain unchanged)

(or both, or something else?)

Reply:
Ah, yes sorry that comment wasn't very clear and I was conflating the host configs with the exposed route. What I was really asking was question (1): should we add an additional `/api/status` endpoint so that when control plane updates all of the health checks they don't need to have any special handling in place for the health gateway?

Reply:
Yeah, I think it makes sense. I will adapt accordingly